benches/benches/set/blockchain.rs

```diff
@@ -320,15 +391,23 @@ pub fn run(c: &mut Criterion) {
     run_group_ref(
         &mut c.benchmark_group("mint"),
         "mint",
-        VmBench::contract(rng, op::mint(RegId::ZERO, RegId::ZERO))
-            .expect("failed to prepare contract"),
+        VmBench::contract_using_db(
+            rng,
+            db.checkpoint().into_vm_db(),
+            op::mint(RegId::ZERO, RegId::ZERO),
+        )
+        .expect("failed to prepare contract"),
     );
 
     run_group_ref(
         &mut c.benchmark_group("burn"),
         "burn",
-        VmBench::contract(rng, op::mint(RegId::ZERO, RegId::ZERO))
-            .expect("failed to prepare contract"),
+        VmBench::contract_using_db(
+            rng,
+            db.checkpoint().into_vm_db(),
+            op::burn(RegId::ZERO, RegId::ZERO),
```

Comment:
It would be nice to burn the asset that exists. Plus we need to burn a `non-zero` value.